src/LostAndFound/Game.cs

```diff
@@ -4,6 +4,8 @@ namespace LostAndFound
 {
     public static class Game
     {
+        private enum MenuChoice { Explore, GatherFood, BuildFortifyCamp, Inventory, MainMenu }
```

Comment:
I think `BuildFortifyCamp` should be `BuildOrFortifyCamp`

Reply:
That is what I intended to do to start with. I committed the change.